pagefetcher.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python2
+
+import urllib
+from os import path, makedirs
+
+"""
+Description:
+
+    This program is a module for wiphisher.py file which includes functionality
+    for downloading extra templates.
+
+Copyright:
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>
+"""
+__author__ = "blackHatMonkey"
+__license__ = "GPL"
+__version__ = "0.1"
+__status__ = "Development"
+__email__ = "dark.world@riseup.net"
+__maintainer__ = "blackHatMonkey"
+
```

Comment:
You should follow the same structure and style as in wifiphisher.py:
* License is in a separate file, no need to be here.
* We do not put any metadata in the code (author, license, etc). 
* Coding style should match. Check for PEP8 fixes.